subt/artf_detector.py

```diff
@@ -0,0 +1,226 @@
+import numpy as np
+import torch
+
+import subt.artf_model
+
+
+class Detector:
+    def __init__(self, model, confidence_thresholds, categories, device,
+                 max_gap, min_group_size):
+        self.model = model
+        self.confidence_thresholds = confidence_thresholds
+        self.categories = categories
+        self.device = device
+        self.max_gap = max_gap
+        self.min_group_size = min_group_size
```

Comment:
I think we want as much determinism as we can get, globally, in any code. Having said that, I don't know either where to put it.

Reply:
My preference is to merge this PR as something what worked well, test that `torch` is not deterministic on collected data and then in new PR add this two lines ... i.e. so that we have something in `master` what is handling new artifacts + is using new robotika base image.